reflect_test.go

```diff
@@ -109,6 +109,10 @@ type TestUser struct {
 	// Test for raw JSON
 	Anything any             `json:"anything,omitempty"`
 	Raw      json.RawMessage `json:"raw"`
+
+	// Test for minLength working for string slice
+	MinLen []string  `json:"min_len" jsonschema:"minLength=2"`
+	MinVal []float64 `json:"min_val" jsonschema:"minimum=2.5"`
```

Comment:
Done in 6f92551.

Additionally, I removed recursive array (`[][]...` case) tags application per the impossibility of making decision where the tags belong to. This might lead to issue with `[][]string `jsonschema:"minLength=1"` not handled properly, but I think it's fine for now.